integrations/jenkins/.port/spec.yaml

```diff
@@ -0,0 +1,24 @@
+type: jenkins
+description: Jenkins integration for Port Ocean
+icon: Jenkins
+features:
+  - type: exporter
+    section: Git Providers
+    resources:
+      - kind: job
+      - kind: build
+configurations:
+  - name: jenkinsHost
+    description: "The host of the Jenkins server"
+    required: true
+    type: url
+  - name: jenkinsUser
+    required: true
+    type: string
+    description: "Jenkins username"
```

Comment:
Please remove the description if its not valuable

Reply:
alright